pkg/certs/util.go

```diff
@@ -0,0 +1,209 @@
+// Copyright 2020 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package cert
```

Comment:
Can we merge this into security/pkg/util/certutil, for better code management?
Also, this code isn't ideal since it has implicit dependency on the k8s secrets. Can we put all the k8s-related stuff in security/pkg/k8s/secret, and even better, try to define a "root-key-cert accessor" interface?

Reply:
+1